src/serializers/GltfSerializer.cpp

```diff
@@ -325,8 +336,12 @@ void GltfSerializer::finalize() {
 	json scene_0;
 	scene_0["nodes"] = node_array_;
 	json_["scenes"].push_back(scene_0);
-	json_["bufferViews"].push_back({ {"buffer", 0}, { "byteLength", indices_length } });
-	json_["bufferViews"].push_back({ {"buffer", 0}, {"byteStride", 12}, { "byteOffset", indices_length },  { "byteLength", binary_length - indices_length } });
+
+	for (auto &n : json_["bufferViews"]) {
+		if (n.contains("byteStride"))
+			n["byteOffset"] = (int)n["byteOffset"] + indices_length;
+	}
```

Comment:
```suggestion
		if (n.contains("byteStride")) {
			n["byteOffset"] = (int)n["byteOffset"] + indices_length;
		}
	}
```

We don't have a proper C++ style guide yet. But this is more inline.

Reply:
ok